### DISCUSSION TITLE
fig-dpi doesn't affect dpi in the revealjs format

### DISCUSSION THREAD

**Discussion by nilsreimer** [quarto-dev/quarto-cli#4438] [Q&A] [answered]
I love using Quarto to put together my slides for teaching statistics—but have one problem I can't figure out.  

I use the `revealjs` format and include figures using the `ggplot2` R package. For example, I might include the following code chunk:

```
#| echo: false
#| fig-width: 4.375
#| fig-dpi: 300
ggplot(data = NULL, aes(x = rnorm(10), y = rnorm(10))) +
  geom_point()
```

I find, however, that the figure in the generated .html presentation still has the default resolution (96 dpi). What am I doing wrong? How can I get a higher-resolution figure? I use RStudio on Windows 11 in case that is relevant.

Thank you in advance.

**Comment by cscheid** [comment, marked as answer]
`fig-dpi` is not currently forwarded as a _cell option_. Try this:

````
---
format: revealjs
title: DPI test
fig-dpi: 300
---

```{r}
#| echo: false
#| fig-width: 4.375
library(ggplot2)
ggplot(data = NULL, aes(x = rnorm(10), y = rnorm(10))) +
  geom_point()
```
````

<img width="2093" alt="image" src="https://user-images.githubusercontent.com/285675/220187610-9854e05a-0fe8-481b-9c5f-4d12721343e7.png">

**Reply by mcanouil** [reply]
If I am not mistaken, one could use the knitr option for code cells, _i.e._, `fig.dpi` instead.
Although, it can easily get confusing.

**Reply by nilsreimer** [reply]
Thank you! This solved my issue.
